src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPresenter.java

```diff
@@ -589,6 +589,8 @@ public void onError(ServerError error)
                session_.getSessionInfo().getActiveProjectDir() != null ?
                session_.getSessionInfo().getActiveProjectDir() :
                workbenchContext_.getCurrentWorkingDir());
+
+         dialog.checkboxExcludeGitIgnore_.setValue(true);
```

Comment:
This is meant to have the "Exclude files matched by .gitignore" set by default if no state is already available. 

Is there another way to do this, with a `json` file or something ?

Reply:
I'm not really sure what this is doing, for me the checkbox was always unchecked. I'm not sure what you're asking about with the json file, you could have a user preference in settings/command palette to set whether it's checked by default, but I'm not sure if that's necessary, I think it makes sense to have the exclude git ignore checked by default, as I don't think most users would have reason to search those files